src/python/phenix_apps/apps/scorch/pcap/pcap.py

```diff
@@ -6,6 +6,16 @@
 from phenix_apps.common import logger, utils
 
 # TODO: bridge capture functionality from mm.py component
+# The issue with bridge captures in phenix is it will capture MGMT traffic,
+# which includes traffic from providers to simulators and bennu field devices.
+# The workaround for this could be to apply a filter with "capture pcap filter"
+# that excludes the IP address ranges used by MGMT and any other undesirable VLANs.
+# Additionally, if multiple experiments are running simultaniously on the same
+# host, then the capture will pick up traffic from those experiments, if they
+# share the same bridge.
+# Perhaps a workaround for this would be to add functionality to minimega to
```

Comment:
Alternatively, you could use the phenix [bridge-mode](https://github.com/sandialabs/sceptre-phenix/blob/main/src/go/cmd/root.go#L204) option, setting it to `auto`, which will put each experiment on its own bridge.